public/styles/HomePage.scss

```diff
@@ -20,13 +20,28 @@
 
     .main {
       #primary {
+        float: left;
+        padding: 0 0 0 2em;
+        text-align: center;
+        width: 78%;
```

Comment:
The properties from line 23 - 26, except text-align have already been set in index.scss lines 45 - 47. So you can remove these.

Reply:
You're right. I've fixed this now